src/core/helpers/page.ts

```diff
@@ -87,12 +87,25 @@ export function createGetServerSideProps(options: CreateGetServerSideProps) {
 
       if (
         !features.some((f) => f.code === "workspaces") &&
-        ctx.resolvedUrl.startsWith("/workspaces")
+        (ctx.resolvedUrl.startsWith("/workspaces") || ctx.resolvedUrl === "/")
       ) {
         return {
           redirect: {
             permanent: false,
-            destination: "/",
+            destination: "/catalog",
+          },
+        };
+      }
+
+      if (
+        features.some((f) => f.code === "workspaces") &&
+        features.some((f) => f.code === "openhexa_legacy") &&
+        ctx.resolvedUrl === "/"
+      ) {
+        return {
+          redirect: {
+            permanent: false,
+            destination: "/catalog",
```

Comment:
User with the feature flag `workspaces` will be redirected to `/workspaces` when he tries to hit `/`